dev.diegovsky.Riff.json

```diff
@@ -0,0 +1,71 @@
+{
+  "app-id": "dev.diegovsky.Riff",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "48",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command": "riff",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--talk-name=org.freedesktop.secrets",
```

Comment:
doesn't use the secrets portal? what are you using to handle this?

Reply:
We're using https://crates.io/crates/secret-service, basically direct raw dbus comms.

Is it okay to let this pass only for this release so we can replace those with the portal implementation? Otherwise this might be delayed and I'm not sure when I'll have the time to work on Riff again.